src/server/pkg/worker/api_server.go

```diff
@@ -257,6 +263,9 @@ func (a *APIServer) runUserCode(ctx context.Context, logger *taggedLogger, envir
 }
 
 func (a *APIServer) uploadOutput(ctx context.Context, tag string, logger *taggedLogger, inputs []*Input) error {
+	defer func(start time.Time) {
+		logger.Logf("uploading output for inputs (%v) took %v\n", inputs, time.Since(start))
+	}(time.Now())
```

Comment:
Same here, no need for logging `inputs`, they're already present in the `LogMessage`